### DISCUSSION TITLE
Embed youtube shorts url

### DISCUSSION THREAD

**Discussion by Yinzy00** [umbraco/Umbraco-CMS#13630] [Features and ideas]
Hi,

When using shorts url in the umbraco Embed I keep getting the "not supported" error.

### Steps to reproduce
1. Open the embed functionality of rich text editor.
2. Use a youtube.com/shorts/.* url.

**Comment by nul800sebastiaan** [comment]
Please make sure to report bugs here: https://github.com/umbraco/Umbraco-CMS/issues/new/choose

I've converted this to an issue now: https://github.com/umbraco/Umbraco-CMS/issues/13632
